at_client/lib/src/manager/monitor.dart

```diff
@@ -84,7 +84,7 @@ class Monitor {
     _regex = monitorPreference.regex;
     _keepAlive = monitorPreference.keepAlive;
     _lastNotificationTime = monitorPreference.lastNotificationTime;
-    _remoteSecondary = RemoteSecondary(atSign, preference);
+    _remoteSecondary ??= RemoteSecondary(atSign, preference);
```

Comment:
if monitor is created with active connection and then internet on the client side drops..the remotesecondary instance that was created initially will be obsolete..right?

Reply:
Any command from remote secondary goes via the sendCommand method in the at_lookup_impl. The sendCommand method creates a new connection whenever a connection is not available.